File: string/1. Two Sum.c
int* twoSum(int* nums, int numsSize, int target) {
  int* ret = (int *)malloc(2 * sizeof(int));
  for (ret[0] = 0; ret[0] < numsSize; ++ret[0]) {
    for (ret[1] = ret[0] + 1; ret[1] < numsSize; ++ret[1]) {
      if (nums[ret[0]] + nums[ret[1]] == target) {
        return ret;
      }
    }
  }
  return ret;
}

File: string/1. Two Sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        memo = {}
        for i in range(len(nums)):
            memo[nums[i]] = i
        for i in range(len(nums)):
            complement = target - nums[i]
            if complement in memo:
                return [i, memo[complement]]